WordPress/src/main/res/values/strings.xml

```diff
@@ -288,6 +288,7 @@
     <!-- post uploads -->
     <string name="upload_failed_param">Upload failed for \"%s\"</string>
     <string name="media_files_not_uploaded">%d media files not uploaded</string>
+    <string name="media_file_not_uploaded">1 media file not uploaded</string>
```

Comment:
I don't have strong feelings about this so we don't have to change it, but it feels wrong to start these sentences with a numeral. How about "Failed to upload media file" and "Failed to upload %d media files" instead?

Reply:
I'm open to changing it - I think maybe let's do an editor copy in the end, with all the notifications in place and merged into #6713 so it's easier for people to weigh in and compare all cases. Sounds good?